tests/generic_config_updater/test_cacl.py

```diff
@@ -2,110 +2,151 @@
 import pytest
 
 from tests.common.helpers.assertions import pytest_assert
-from tests.common.config_reload import config_reload
 from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_res_success, expect_op_failure
 from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+# Test on t0 topo to verify functionality and to choose predefined variable
+# admin@vlab-01:~$ show acl table
+# Name        Type       Binding          Description    Stage
+# ----------  ---------  ---------------  -------------  -------
+# ...
+# NTP_ACL     CTRLPLANE  NTP              NTP_ACL        ingress
+# SNMP_ACL    CTRLPLANE  SNMP             SNMP_ACL       ingress
+# SSH_ONLY    CTRLPLANE  SSH              SSH_ONLY       ingress
 
 pytestmark = [
-    pytest.mark.topology('any'),
+    pytest.mark.topology('t0'),
```

Comment:
>t0 [](http://example.com/codeflow?start=26&length=2)

It is not limited to t0.

Reply:
My previous test is to remove all ACL_TABLE and ACL_RULE settings, then test based on that. So it could be applied to any topo.
You mentioned we better test the scenario more practically. So I think it is more applicable that users just modify the change based on one of the topo.
That's why I change to `t0` and just make changes above that.
Besides, rollback verification is easier if we have known the topo.

Reply:
I see. Then I feel there is also practical test case for t1. Let's discuss and implement in another PR.